Python/ceval.c

```diff
@@ -2683,7 +2683,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
         case TARGET(STORE_NAME): {
             PyObject *name = GETITEM(names, oparg);
             if (name == NULL) {
-                name = _PyHydrate_LoadName(co->co_pyc, oparg);
+                name = _PyHydrate_LoadName(co->co_pyc, co->co_strings_start + oparg);
```

Comment:
Probably this entire block should be a macro. Macros that contain a `goto` are used elsewhere, e.g. `DEOPT_IF()`.

Reply:
Yes, that’s another reason to use a macro :-)